src/test/java/org/tugraz/sysds/test/functions/lineage/FullReusePerfTest.java

```diff
@@ -90,7 +90,9 @@ public void testLineagePerf(String testname) {
 			writeInputMatrixWithMTD("X", X, true);
 			
 			Lineage.resetInternalState();
+			OptimizerUtils.setLinResFull();
```

Comment:
maybe call it LinReuse or even LineageReuse to make clear what this method is supposed to do.